lemminx-maven/src/main/java/org/eclipse/lemminx/maven/SubModuleValidator.java

```diff
@@ -32,23 +33,19 @@ public void setPomFile(File pomFile) throws FileNotFoundException, IOException,
 		model = mavenreader.read(new FileReader(pomFile));
 	}
 
-	public Diagnostic validateSubModuleExistence(DiagnosticRequest diagnosticRequest) {
+	public Optional<Object> validateSubModuleExistence(DiagnosticRequest diagnosticRequest) {
```

Comment:
Why not `Optional<List<Diagnostic>>` here?

Reply:
Oops, good catch.